### PR TITLE
implement LocalStateFinder#hasDiffBetweenOriginAndHead

### PR DESCRIPTION
**Pull request by 1natsu172**
## Overview
Implemented a method to determine the difference between Origin and Head.
This is useful for determining if there is data that has not been pushed yet.

### CODE REVIEW

File: modules/redux/test/integration.test.ts [hunk 1]
```diff
@@ -242,4 +242,46 @@ describe("Integration", () => {
     const { entities } = store.getState().phenyl;
     assert.deepStrictEqual(entities, {});
   });
+
+  it("should be check for diff Origin and Head", () => {
```
Comment (shinout) [inline modules/redux/test/integration.test.ts]: Better expression for this wil be `should check diff between origin and head in EntityInfo`.

Comment (1natsu172) [inline modules/redux/test/integration.test.ts]:
```suggestion
  it("should check diff between origin and head in EntityInfo", () => {
```

Comment (1natsu172) [reply modules/redux/test/integration.test.ts]: Thank you. I applied the suggested expression.

**Comment by shinout** [comment]
LGTM too.
@sasurau4 Can you merge this PR?

**Pull request merged**